routers/web/user/home.go

```diff
@@ -746,7 +756,8 @@ func getRepoIDs(reposQuery string) []int64 {
 		return []int64{}
 	}
 	if !issueReposQueryPattern.MatchString(reposQuery) {
-		log.Warn("issueReposQueryPattern does not match query")
+		// FIXME: sometimes it could be "[W] issueReposQueryPattern [4%2C5%2C7%2C6%2C8%2C] does not match query"
```

Comment:
Please help fix it if anyone has any ideas.

Reply:
I also see this many times, IIRC it is caused by incorrect template parameter escaping (double-encoding).

And, passing query parameters by "[1,23]" is quite hacky, but fixing it needs a non-trivial refactoring.

Reply:
The bug is here, it should use `,` but not `%2C` (ps: although I have touched the code, but it wasn't written by me 😂)

![image](https://github.com/go-gitea/gitea/assets/2114189/f1558fd5-7f78-4196-bcaa-66143ddda7df)

Reply:
👍  Fixed in https://github.com/go-gitea/gitea/pull/26556